ux.symfony.com/src/Twig/MarkdownExtension.php

```diff
@@ -0,0 +1,48 @@
+<?php
+
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace App\Twig;
+
+use App\Service\CookBookParser;
+use Twig\Environment;
+use Twig\Error\LoaderError;
+use Twig\Error\RuntimeError;
+use Twig\Error\SyntaxError;
+use Twig\Extension\AbstractExtension;
+use Twig\TwigFunction;
+
+class MarkdownExtension extends AbstractExtension
+{
+    public function __construct(
+        private readonly CookBookParser $cookBookParser,
```

Comment:
A class named "MarkdownExtension" should probably have no knowledge of the Cookboor :)